### PR TITLE
Create October CMS Stack Pack

### PR DESCRIPTION
**Pull request by ayumi-cloud**
## Submitting a new stack pack

Want to submit a brand new stack pack? Awesome! In order for a stack pack to be considered however, the following must be met:

- There should be a simple way to detect the stack via JS/DOM properties.

October CMS can be detected by many popular software programs out there, some examples:

https://trends.builtwith.com/cms/October-CMS
https://github.com/AliasIO/wappalyzer/pull/3022
etc.

Popular software programs detect October using various methods such as:

Meta-tag (generator): "OctoberCMS"
Cookie Name: "october_session"
Javascript Framework Detection: 

```
/modules/system/assets/css/framework.extras-min.css
/modules/system/assets/js/framework.combined-min.js
/modules/system/assets/css/framework.extras-min.css
/modules/system/assets/js/framework.combined-min.js
```

Finding one of these files.

etc.

(There are many ways to detect this cms in the wild).

- At least 6 of the performance audits should have stack-specific recommendations.

See commit, we have done that.

- A significant number of websites (~ in the 10's of thousands) should be using the stack. Consider using [HTTP Archive](https://discuss.httparchive.org/t/javascript-library-detection/955/2) to detect the number sites used by the particular tool.

image

Yes, October CMS is used by 10's of thousands of websites.

- There must be a minimum of two maintainers for each stack from the community who can help review and provide suggestions. We (Google) should not be the ones to decide what the audit recommendations should be.

Please see this link: https://github.com/octobercms/october#development-team

There are currently: 2 creators and 4 maintainers, plus many other people helping the project on a regular basis.

If all of these are met, submit a PR for review! Copy the `general.json` file into the `packs/` directory and change each audit string to a specific recommendation regarding your stack. If you can't provide a recommendation for a particular audit, change it to an empty string. However, you will need to provide at least 6 audit strings as mentioned earlier.

### CODE REVIEW

**Comment by ayumi-cloud** [comment]
@connorjclark @patrickhulce @LukeTowers

File: packs/october.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  /** Additional description of a Lighthouse audit that tells the user how they can improve site performance by optimizing images, in the context of the October CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'uses-optimized-images': "Consider using an [image optimization plugin](https://octobercms.com/plugins?search=image) to compresses images while retaining the quality.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance via enabling text compression in the context of the October CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'uses-text-compression': "Enable text compression in the web server configuration.",
```
Comment (housseindjirdeh) [inline packs/october.js:27]: Nit: If there's a link to a doc page for this that can be included, that would be great :)

File: packs/october.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  'unminified-javascript': "There are many [plugins](https://octobercms.com/plugins?search=javascript) that can speed up a website by concatenating, minifying and compressing the scripts. Using a build process to do this minification up-front can speed up development.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by encoding animated images as video, in the context of the October CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'efficient-animated-content': "[Replace animated GIFs with video](https://web.dev/replace-gifs-with-videos/) for faster web page loads and consider using modern file formats such as [WebM](https://web.dev/replace-gifs-with-videos/#create-webm-videos) or [AV1](https://developers.google.com/web/updates/2018/09/chrome-70-media-updates#av1-decoder) to improve compression efficiency by greater than 30% over the current state-of-the-art video codec, VP9.",
```
Comment (housseindjirdeh) [inline packs/october.js:19]: The audit suggestion here is great, but doesn't include any specific advice for October CMS. Maybe we can remove it entirely?

**Review by housseindjirdeh — APPROVED** [review]
All in all looks great, just a few tiny nits. Thanks so much for landing this :)

In the interest of time, I may merge this asap so we can have this in the next Lighthouse release. A future PR might be needed to resolve these tiny nits if that's okay.

File: packs/october.js (hunk 1 above)
Comment (ayumi-cloud) [reply packs/october.js:27]: Updated the cms doc's here: https://github.com/octobercms/docs/pull/501

When merged will update the pr in this repo.

File: packs/october.js (hunk 2 above)
Comment (ayumi-cloud) [reply packs/october.js:19]: Updated the cms doc's here: https://github.com/octobercms/docs/pull/501

When merged will update the pr in this repo.

**Pull request merged**